addons/website/static/tests/website_tests.js

```diff
@@ -6,6 +6,7 @@ var testUtils = require("web.test_utils");
 
 var createView = testUtils.createView;
 
+// TODO: remove me when legacy redirect button is removed
```

Comment:
Strange, we already had a "redirect_field_tests.js" file alongside a "website_tests.js" file testing only redirect buttons? :laughing: 

Note: even more reason to do the thing I said and have both website_redirect_button and website_redirect_button_ledacy, I am sure we will forget about this file otherwise ^^